.travis.yml

```diff
@@ -23,8 +23,8 @@ addons:
 env:
   global:
   - R_BUILD_ARGS="--no-build-vignettes --no-manual"
-- R_CHECK_ARGS="--no-build-vignettes --no-manual --as-cran"
-- _R_CHECK_FORCE_SUGGESTS_=false
+  - R_CHECK_ARGS="--no-build-vignettes --no-manual --as-cran"
+  - _R_CHECK_FORCE_SUGGESTS_=false
 
 warnings_are_errors: true
 
```

Comment:
check below...you don't want `travis_wait 20 R CMD check dataRetrieval*tar.gz`

Reply:
😬

Reply:
up above there are indentation errors too in the matrix: